packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -444,8 +446,15 @@ const KMS = WebexPlugin.extend({
             this.logger.info('kms: request error', reason.stack || reason);
           }
 
+          consoleDebug(`timeout ${timeout}`);
           timeout *= 2;
 
+          if (timeout >= this.config.ecdhMaxTimeout) {
+            this.logger.info('kms: exceeded maximum KMS request retries');
+
+            return Promise.reject(reason);
+          }
+
```

Comment:
Will this condition ever be met? If the timeout keeps getting doubled it will go to 6, 12, 24, 48, which is still less than 96, so it will skip this, but it'll meet the condition below as it's greater than 32, then get set back to 0 and start all over again?